tools/stress-cluster/chaos/README.md

```diff
@@ -411,25 +407,52 @@ src/
   scenarioLongRunning.js
   scenarioPeekMessages.js
   scenarioBatchReceive.js
+Dockerfiles/
+  DockerfileLR
+  DockerFilePM
+  DockerfileBR
 ...
 ```
 
 The pod command in the job manifest could be configured to run a variable file name:
 
 ```
-command: ["node", "app/{{ .Stress.Scenario }}.js"]
+command: ["node", "app/{{ .Stress.testTarget }}.js"]
 ```
 
-In order to accomplish this, add the scenarios list to `values.yaml`
+While the [stress test deploy script](https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/scripts/stress-testing/deploy-stress-tests.ps1) translates and uploads the dockerimage with an image tag that could be referenced in the job manifest:
 
 ```
-scenarios:
-  - scenarioLongRunning
-  - scenarioPeekMessages
-  - scenarioBatchReceive
+image:  {{ .Stress.imageTag }}
 ```
 
-The underlying `stress-test-addons` helm library will handle a scenarios list automatically, and deploy multiple instances of the stress test job, one for each scenario.
+In order to accomplish this, add the configuration to `scenarios-matrix.yaml`
+
+```
+matrix:
+  image:
+    - Dockerfiles/DockerfileLR
+    - Dockerfiles/DockerfilePM
+    - Dockerfiles/DockerfileBR
+  scenarios:
+    LongRunning:
+      testTarget: scenarioLongRunning
+    PeekMessages:
+      testTarget: scenarioPeekMessages
+    BatchReceive:
+      testTarget: scenarioBatchReceive
+```
+
+The [`deploy-stress-tests.ps1`](https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/scripts/stress-testing/deploy-stress-tests.ps1) script will generate a generatedValues.yaml file which contains the scenarios matrix that lists out all the custom configuration for each test instance.
```

Comment:
It would be good to show a quick snippet of what the output generatedValues.yaml file looks like. As well as documenting how to test the matrix locally via the independent script.